components/brave_rewards/resources/ui/components/listToken/style.ts

```diff
@@ -23,13 +23,13 @@ export const StyledTitle = styled<{}, 'div'>('div')`
   color: #4b4c5c;
   flex-grow: 1;
   flex-shrink: 1;
-  flex-basis: 60%;
+  flex-basis: 40%;
```

Comment:
total of this and bottom one is not 100%. Is this intentionally?

Reply:
Not using the entire span was the only way to fit both item lables and amounts in the case of both BAT and BAT points